src/components/horizontal-term-line/HorizontalTermLine.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+export interface HorizontalTermLineProps {
+  title: string;
+  subTitle?: string;
+  linkList: string[];
+}
+
+const HorizontalTermLine: React.FC<HorizontalTermLineProps> = ({
+  title,
+  subTitle,
+  linkList
+}) => {
+  return (
+    <div className="text-small-caption horizontal-term-line">
+      <p className="text-label-semibold">
+        {`${title}`}{" "}
+        {subTitle && <span className="text-small-caption">{subTitle} </span>}
+      </p>
+      <ul className="horizontal-term-line__list">
+        {linkList?.map((item) => (
```

Comment:
linkList has the type: `string[]` and is required. So no optional chaining check is needed here.